### PR TITLE
responsive issue

### LINKED ISSUE #101: Thank you Email
Hi Team, 

We want automatic email to be sent to all contact form users. Email design and content will be same as a thank you page design.

### PR DESCRIPTION
**Pull request by 2016DigiTech**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #101

Test URLs:
- Before: https://main--tlcgrp--hlxsites.hlx.page/
- After: https://responsive-issue--tlcgrp--hlxsites.hlx.page/

### CODE REVIEW

**Comment by 2016DigiTech** [comment]
Hi rofe, reponsiveissue PR

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -429,7 +429,8 @@
     }
   
     main .cards-container .default-content-wrapper h3 {
-      font-size: 28px;
+      font-size: 24px;
+      margin-top:35px;
```
Comment (rofe) [inline blocks/cards/cards.css:433]: use consistent style
```suggestion
      margin-top: 35px;
```

File: blocks/z-pattern/z-pattern.css [hunk 2]
```diff
@@ -241,6 +241,7 @@
 
   .z-pattern-container .default-content-wrapper h4 {
     font-size: 28px;
+    margin-top:35px;
```
Comment (rofe) [inline blocks/z-pattern/z-pattern.css:244]:
```suggestion
    margin-top: 35px;
```

File: styles/styles.css [hunk 3]
```diff
@@ -581,6 +593,8 @@ main .why-us .icon-arrow-about-cta {
   padding-left: 15px;
 }
 
+
+
```
Comment (rofe) [inline styles/styles.css:597]: avoid unnecessary whitespace
```suggestion
```

**Review by rofe — APPROVED** [review]
Looking good, apart from a few minor coding style comments.

**Comment by rofe** [comment]
I noticed that the cards at the bottom of the page have uneven bottom spacing in the mobile viewport:
image
This is also the case in `main`, therefore not related to changes in this PR. I just thought I'd mention it so you can fix it along with the other responsive issues.
